docs/intro/installation.rst

```diff
@@ -96,6 +100,8 @@ Python:
     as this is required for subsequent installation steps and interactive use later on.
     Other than that, using the default installation settings is just fine.
 
+    If you did everything right, typing ``python --version`` in the command prompt should give you an answer.
```

Comment:
Here you could mention the kind of correct answer (instead of an "answer" like "command not found"), e.g. "should show you your installed Python version, like 'Python 3.x.y'."